### PR TITLE
Refactors

### PR DESCRIPTION
**Pull request by xtrycatchx**
minor update to readme, small steps for those who wishes to contribute

### CODE REVIEW

File: src/controllers/mappers/ScanResult.js [hunk 1]
```diff
@@ -2,70 +2,28 @@ import { cleanObject } from '../../common/utils'
...
-            time: req.location.mTime
-        }
-    }
-    if ("networkInfo" in req) {
-        var cellTower = {
-            cid: req.networkInfo.cid,
-            lac: req.networkInfo.lac,
-            mcc: req.networkInfo.mcc,
-            mnc: req.networkInfo.mnc,
+const _transform = object => [object].reduce((acc, cur, i) => {
```
Comment (TheoryOfNekomata) [inline src/controllers/mappers/ScanResult.js]: I suggest we use a different name for this function.

File: src/domain/repository/cellTowerRepository.js [hunk 2]
```diff
@@ -44,4 +44,4 @@ export class CellTowerRepository {
     }
 }
 
-const shouldLookup = (result) => null == result || result.status == 'Pending'
\ No newline at end of file
+const shouldLookup = (result) => !result || result.status == 'Pending'
```
Comment (TheoryOfNekomata) [inline src/domain/repository/cellTowerRepository.js:47]: A better construct is advised for checking existence.

For instance, `[] == false` evaluates to `true`.

With this, strict equality operators (`===` and `!==`) perform more consistently than their non-strict counterparts.

**Comment by TheoryOfNekomata** [comment]
Please consider the comments added for better code quality.

File: src/controllers/mappers/ScanResult.js (hunk 1 above)
Comment (xtrycatchx) [reply src/controllers/mappers/ScanResult.js]: I agree. However, it was decided to not do reducing and just use imperative retrival of objects from request as suggested by @melbystyle . So for now, i reverted to original approach with a few changes

File: src/domain/repository/cellTowerRepository.js (hunk 2 above)
Comment (xtrycatchx) [reply src/domain/repository/cellTowerRepository.js:47]: I agree

File: src/controllers/mappers/ScanResult.js (hunk 1 above)
Comment (TheoryOfNekomata) [reply src/controllers/mappers/ScanResult.js]: Understood. Will wait for new commits.

**Pull request merged**